tools/Makefile

```diff
@@ -38,13 +38,7 @@ SET_VERSION_SCRIPTS = \
 	bin/gpssh-exkeys \
```

Comment:
Why not remove gpssh-exkeys as we have hawq ssh now?
Why not remove clsAddMirrors.py as we don't have mirrors in hawq?